bundle/python_bundle_workflow/scripts/inference.py

```diff
@@ -63,7 +63,6 @@ def __init__(self, dataset_dir: str = "."):
             nib.save(n, os.path.join(dataset_dir, f"seg{i:d}.nii.gz"))
 
         self._props = {}
-        self._set_props = {}
         self.dataset_dir = dataset_dir
 
     def initialize(self):
```

Comment:
This should also set `self._props`?

Reply:
I have removed this line, as it appears to overwrite the probs that have already been set in the init.

Reply:
I have added back and updated to set `self._props`.